SDK v2 migration guide.md

```diff
@@ -242,10 +246,10 @@ What was a loose affiliation of separate clients is now a consolidated client wi
 | `Module.LastActivityTime` | `Module.LastActiveOnUtc` | See¹ |
 | `RegistryManager.GetTwinAsync(...)` | `IotHubServiceClient.Twins.GetAsync(...)` | |
 | `RegistryManager.UpdateTwinAsync(...)` | `IotHubServiceClient.Twins.UpdateAsync(...)` | |
-| `Twin` | `ClientTwin` | "Client" is a word we often use to indicate the device- or module-side of the data-plane.² |
+| `Twin` | `TwinProperties` | The device only gets properties, not the full twin. |
 | `Twin.StatusUpdatedOn` | `ClientTwin.StatusUpdatedOnUtc` | See¹ |
 | `Twin.LastActivityOn` | `ClientTwin.LastActiveOnUtc` | See¹ |
-| `TwinCollection` | `ClientTwinProperties` | See² |
+| `TwinCollection` | `ClientTwinProperties` | "Client" is a word we often use to indicate the device- or module-side of the data-plane.² |
```

Comment:
is the dash meant to be here? After `device`

Reply:
Yes, because device-side is a suffixed word, and we're sharing the suffix with module-.

I think this is considered proper English, but I could be wrong.